include/rdma/fabric.h

```diff
@@ -178,6 +178,7 @@ enum {
 	FI_ADDR_MLX,
 	FI_ADDR_STR,		/* formatted char * */
 	FI_ADDR_PSMX2,		/* uint64_t[2] */
+	FI_ADDR_VERBS_UD,
```

Comment:
I think this needs to be IB_UD instead of verbs.  It's an IB specific address.  Someday verbs may have non-IB UD support.